### ISSUE TITLE
Support for Cloud Development Kit

### ISSUE THREAD

**Issue by neonelemental** [boltops-tools/jets#544]
# TLDR

I'm enquiring about something I was unable to find in the docs but which I thought you may be able to provide more context on.  I am new to Lambda and I want to determine whether what I want can/should be integrated into the framework, or whether what I want to do is already doable within the framework.

More than happy to flesh this out but for now I don't have enough information.

## Summary

I have a CDK repository I would like to make use of when deploying a jets application to AWS.  I'm new to lambda so I might just need some direction, but feature request seemed to be the best category for what I am enquiring about.

## Motivation

Our cloud formation provisioning processes are backed by a CDK repository.

## Guide-level explanation

- Given I have an existing jets app, how can I provision lambda using CDK.

**Comment by tongueroo** [comment]
Thanks for in enquiry. CDK is its own different beast and has its own deployment package procedure. Current thoughts on this is that it'll be overly complex and is out of scope.
